models/bg.model.js

```diff
@@ -69,16 +69,34 @@ exports.addReviewComment = (review_id, reqBody) => {
         }
         return db
             .query(
-            `INSERT INTO comments
+                `INSERT INTO comments
             (author, body, review_id)
             VALUES ($1, $2, $3)
             RETURNING *;`,
                 [username, body, review_id]
             )
-            .then((comment) => { 
+            .then((comment) => {
                 return comment.rows;
             });
     } else {
         return Promise.reject({ status: 400, msg: "Invalid comment" });
     }
 };
+
+exports.updateReviewById = (review_id, inc_votes) => {
+    const votes = inc_votes
+
+    if (!votes) {
```

Comment:
Be careful here - this would trigger if votes was 0 - although strange should still work.
You may want to check if votes is explicitly undefined